sagemaker_neo_compilation_jobs/tensorflow2_resnet/sagemaker-neo-tf-resnet.ipynb

```diff
@@ -0,0 +1,569 @@
+{
```

Comment:
<p>Accessibility: use page title ("imagenet/resnet_v2_50/classification") or similar for link text instead of "here"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3207/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>